### ISSUE TITLE
Uncaught TypeError: Cannot read property 'setStatusLoaded' of null

### ISSUE THREAD

**Issue by twopoint718** [idris-hackers/atom-language-idris#67]
[Enter steps to reproduce below:]

1. I launched atom with `atom bounds.idr`
2. Ran the typechecker with ctrl+option+R.

**Atom Version**: 1.0.15
**System**: Mac OS X 10.10.5
**Thrown From**: [language-idris](https://github.com/idris-hackers/atom-language-idris) package, v0.3.0

### Stack Trace

Uncaught TypeError: Cannot read property 'setStatusLoaded' of null

```
At /Users/chris/.atom/packages/language-idris/node_modules/rx-lite/rx.lite.js:549

TypeError: Cannot read property 'setStatusLoaded' of null
    at AnonymousObserver._onNext (/Users/chris/.atom/packages/language-idris/lib/idris-controller.coffee:128:32)
    at AnonymousObserver.Rx.AnonymousObserver.AnonymousObserver.next (/Users/chris/.atom/packages/language-idris/node_modules/rx-lite/rx.lite.js:1811:12)
    at AnonymousObserver.Rx.internals.AbstractObserver.AbstractObserver.onNext (/Users/chris/.atom/packages/language-idris/node_modules/rx-lite/rx.lite.js:1743:35)
    at AnonymousObserver.tryCatcher (/Users/chris/.atom/packages/language-idris/node_modules/rx-lite/rx.lite.js:537:29)
    at AutoDetachObserverPrototype.next (/Users/chris/.atom/packages/language-idris/node_modules/rx-lite/rx.lite.js:5421:51)
    at AutoDetachObserver.Rx.internals.AbstractObserver.AbstractObserver.onNext (/Users/chris/.atom/packages/language-idris/node_modules/rx-lite/rx.lite.js:1743:35)
    at FilterObserver.onNext (/Users/chris/.atom/packages/language-idris/node_modules/rx-lite/rx.lite.js:4114:34)
    at FilterObserver.tryCatcher (/Users/chris/.atom/packages/language-idris/node_modules/rx-lite/rx.lite.js:537:29)
    at AutoDetachObserverPrototype.next (/Users/chris/.atom/packages/language-idris/node_modules/rx-lite/rx.lite.js:5421:51)
    at AutoDetachObserver.Rx.internals.AbstractObserver.AbstractObserver.onNext (/Users/chris/.atom/packages/language-idris/node_modules/rx-lite/rx.lite.js:1743:35)
```

### Commands

```
     -0:02.3.0 language-idris:typecheck (atom-text-editor.editor.is-focused)
```

### Config

```json
{
  "core": {
    "themes": [
      "atom-dark-ui",
      "monokai"
    ],
    "ignoredNames": [
      ".git",
      ".svn",
      ".DS_Store"
    ]
  },
  "language-idris": {}
}
```

### Installed Packages

```coffee
# User
asciidoc-preview, v0.6.0
language-asciidoc, v0.12.0
language-idris, v0.3.0
monokai, v0.18.0

# Dev
No dev packages
```

**Comment by archaeron** [comment]
thanks for the report. I'm removing the statusbar, since it doesn't really work as it should anyway.

**Comment by archaeron** [comment]
https://github.com/idris-hackers/atom-language-idris/commit/0f75e2accb6c45ce1bb396f3873d4df7fef75c04

will be fixed in the next version :+1:

**Comment by archaeron** [comment]
published the nev version (0.3.1) please reopen the issue persists

**Comment by twopoint718** [comment]
Thanks!
